cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -275,3 +278,41 @@ def test_replication_connect_to_vm_in_region(provider,
         view.login.click()
         view = vm.create_view(InfraVmDetailsView)
         wait_for(lambda: view.is_displayed, message="Wait for VM Details page")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.ignore_stream("upstream")
+@pytest.mark.meta(automates=[1796681])
+def test_appliance_replicate_remote_down(temp_appliance_preconfig_funcscope_rhevm,
+       temp_appliance_unconfig_funcscope_rhevm):
+    """Test that the Replication tab displays in the global appliance UI when the remote appliance
+    database cannot be reached.
+
+    Bugzilla:
+        1796681
+
+    Metadata:
+        test_flag: replication
+
+    Polarion:
+        assignee: tpapaioa
+        initialEstimate: 1/4h
+        casecomponent: Appliance
+    """
+    remote_appliance = temp_appliance_preconfig_funcscope_rhevm
+    global_appliance = temp_appliance_unconfig_funcscope_rhevm
+
+    configure_replication_appliances(remote_appliance, global_appliance)
+
+    global_region = global_appliance.server.zone.region
+    assert global_region.replication.get_replication_status(host=remote_appliance.hostname), (
+        "Remote appliance not found on Replication tab after initial configuration.")
+
+    result = global_appliance.ssh_client.run_command(
+        f"firewall-cmd --direct --add-rule ipv4 filter OUTPUT 0 -d {remote_appliance.hostname}"
+        " -j DROP")
+    assert result.success, "Could not create firewall rule on global appliance."
+
+    global_appliance.browser.widgetastic.refresh()
```

Comment:
Why do we need a browser refresh here? If it is necessary, can you just do `global_appliance.browser.refresh()`?

Reply:
I'm using a browser refresh because the Replication tab is already displayed from the first call to get_replication_status(). After the firewall rule is added, we want to verify that the Replication tab re-displays successfully, but the second call to get_replication_status() doesn't do any navigation/refresh since the view is already being displayed.

global_appliance.browser is a ViaUI instance and doesn't have the refresh() method, whereas global_appliance.browser.widgetastic is the MiqBrowser instance that provides access to refresh() and other Selenium actions.

Reply:
I see, thanks for the explanation. One possible enhancement could be to have a `force` arg or kwarg in that `get_replication_status` so you can refresh the page via that method. I'd consider this _optional_ however.